lib/id3c/cli/redcap.py

```diff
@@ -173,20 +185,20 @@ def _fetch(self, content: str, parameters: Dict[str, str] = {}) -> Any:
 
         headers = {
             'Content-type': 'application/x-www-form-urlencoded',
-            'Accept': 'application/json'
+            'Accept': 'application/json, */*;q=0.1'
         }
```

Comment:
What does `*/*;q=0.1` do here?

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/Quality_values

Reply:
Worth noting that this Accept header is really more descriptive than functional, as I don't think REDCap actually cares about it (unlike other APIs and servers which do). I am realizing that I should probably change it to:

     Accept: application/json, text/*

which more accurately reflects the way the response is handled. (Or just remove it entirely.)

Reply:
Repushed adjusting the Accept header:

```
tom@whunk id3c (redcap-fields-and-version<>) $ git diff-rebase u
1:  39a12c1 = 1:  39a12c1 redcap: Support limiting the record fields fetched
2:  529a09a ! 2:  aa402b7 redcap: Support fetching the REDCap version
    @@ lib/id3c/cli/redcap.py: class Project:
              headers = {
                  'Content-type': 'application/x-www-form-urlencoded',
     -            'Accept': 'application/json'
    -+            'Accept': 'application/json, */*;q=0.1'
    ++            'Accept': 'application/json' if format == "json" else 'text/*'
              }
      
              data = {

```

I think this is clearer and more accurate, so thanks for making me think about it again. :-)